.github/workflows/action-format.yml

```diff
@@ -0,0 +1,130 @@
+name: Format
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  format:
+    name: 'Format JSON'
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/format')
+    steps:
+      - name: 'Post acknowledgement that it will format code'
+        continue-on-error: true # Never fail the build if this fails
+        uses: actions/github-script@e3cbab99d3a9b271e1b79fc96d103a4a5534998c
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'The "Format code" action has started running.'
+            })
+
+      - name: 'Download PR data'
+        run: |
+          PR_DATA="/tmp/pr.json"
+
+          jq -r ".issue.pull_request.url" "$GITHUB_EVENT_PATH" | \
+            xargs curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -o "$PR_DATA" --url
+
+      - name: 'Check fork status'
+        id: fork_status
+        run: |
+          IS_FORK="$(jq '.head.repo.fork' "/tmp/pr.json")"
+          echo "::set-output name=fork::$IS_FORK"
+
+      - name: 'Setup SSH deploy key'
+        if: steps.fork_status.outputs.fork == 'false'
+        run: |
+          mkdir ~/.ssh
+          echo "${{ secrets.DEPLOY_KEY }}" > ~/.ssh/id_ed25519
+          chmod 600 ~/.ssh/id_ed25519
+
+      - name: 'Checkout code'
+        run: |
+          PR_DATA="/tmp/pr.json"
+
+          HEAD_REF=$(jq -r ".head.ref" "$PR_DATA")
+
+          if [ ${{ steps.fork_status.outputs.fork }} == "false" ]; then
+            echo "::debug::Setting up repo using SSH"
+            HEAD_REPO=$(jq -r '.head.repo.ssh_url' "$PR_DATA")
+          else
+            echo "::debug::Setting up repo using HTTPS"
+            HEAD_REPO=$(jq -r '.head.repo.clone_url | sub("https://"; "https://x-access-token:${{ secrets.GITHUB_TOKEN }}@")' "$PR_DATA")
+          fi
+
+          git clone $HEAD_REPO .
+          git checkout -b "$HEAD_REF" "origin/$HEAD_REF"
+
+      - name: Use Node.js LTS (16.x)
+        uses: actions/setup-node@1f8c6b94b26d0feae1e387ca63ccbdc44d27b561
```

Comment:
Any updates needed for `.github/dependabot.yml` for the `uses` actions (such as this one, actions/cache, actions/github-script) ?

Reply:
No, this should be covered by https://github.com/exercism/problem-specifications/blob/main/.github/dependabot.yml#L6. 
See https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems